src/drivers/distance_sensor/lightware_sf45_serial/lightware_sf45_serial.cpp

```diff
@@ -69,15 +68,19 @@ SF45LaserSerial::SF45LaserSerial(const char *port, uint8_t rotation) :
 	}
 
 	_num_retries = 2;
-	_px4_rangefinder.set_device_id(device_id.devid);
-	_px4_rangefinder.set_device_type(DRV_DIST_DEVTYPE_LIGHTWARE_LASER);
 
 	// populate obstacle map members
 	_obstacle_map_msg.frame = obstacle_distance_s::MAV_FRAME_BODY_FRD;
-	_obstacle_map_msg.increment = 5;
-	_obstacle_map_msg.angle_offset = 2.5;
-	_obstacle_map_msg.min_distance = UINT16_MAX;
+	_obstacle_map_msg.sensor_type = obstacle_distance_s::MAV_DISTANCE_SENSOR_LASER;
+	_obstacle_map_msg.increment = 10;
+	_obstacle_map_msg.min_distance = 20;
 	_obstacle_map_msg.max_distance = 5000;
+	_obstacle_map_msg.angle_offset = 0;
```

Comment:
This should be non-zero. There has to be some offset since the obstacle distance array element 0 starts and moves clockwise, however with an increment of 10 degrees there won't be coverage immediately to the left of the actual obstacle distance data

Reply:
I did a quick test where I set the fov of a 1D sensor pointing to the front to be 5 deg.
![image](https://github.com/user-attachments/assets/efc16f23-0379-4945-ad43-3b394ddfe732)
and then the adjacent bins are empty and only the bin with index 0 is populated. Which would support the statement above.

Reply:
Maybe a better test of this is a 1D sensor 20 degree field of view sensor forward facing. If 20 degrees is split down the middle would the bins occupied be 0, 1 and 71, 72? (assuming 5 degree increment).

I guess this could be changed and defined to include that wrapping effect: the angle offset is super non-intuitive to most users, specifically to hose implementing new drivers where there is sparse info on the sign of the angle and what it means.

 It doesn't make a huge difference on the rotating lidars since the degree resolution is small, but would make a difference on 1D lidars with a significant field of view.

Reply:
- [ ] https://github.com/PX4/PX4-user_guide/pull/3364

i added documentation of the angles in the CP Docs update. 
i've also expanded the tests and code of the Collision prevention logic which adds distance sensors and obstacle_distance to the internal map. offset issues should now all be fixed.